msgpack-core/src/main/java/org/msgpack/core/MessagePack.java

```diff
@@ -154,6 +162,7 @@ public int getPackerRawDataCopyingThreshold()
         private int stringEncoderBufferSize = 8192;
         private int stringDecoderBufferSize = 8192;
         private int packerBufferSize = 8192;
+        private int packerLargeStringLengthThreshold = 512;
```

Comment:
This parameter doesn't tell how this threshold value will be used. How about `packerSmallStringOptimizationThreshold`? And also add a comment `this parameter is subject to change` so that we can fix the optimization in future.